services/cache-service-template.yaml

```diff
@@ -4,14 +4,11 @@ labels:
   template: cache-service
 metadata:
   annotations:
-    description: Red Hat Data Grid is a high performance, linearly scalable, key/value
-      data grid solution. It provides many features to serve a variety of use cases.
+    description: Red Hat Data Grid is an in-memory, distributed key/value store.
     iconClass: icon-datagrid
     openshift.io/display-name: Red Hat Cache Service
     openshift.io/documentation-url: https://github.com/jboss-container-images/jboss-datagrid-7-openshift-image/blob/datagrid73-dev/docs/cache-service.asciidoc
-    openshift.io/long-description: |-
-      In this service, Red Hat Data Grid is configured as an in-memory caching service without persistence. Data in the cache will be lost if the service is stopped or restarted, and therefore, it should not be used as a permanent store for data.
-      It can be used to improve your application’s response time, by keeping frequently and recently accessed data in a high speed in-memory store.
+    openshift.io/long-description: Red Hat Data Grid Cache Service provides an opinionated implementation of Data Grid for OpenShift for improving application speed and performance.
```

Comment:
Not sure about the use of "opinionated" here. Doesn't it have slightly negative connotation?

Reply:
I've changed to minimal configuration. thanks.